src/view/frontend/templates/react-script.phtml

```diff
@@ -6,8 +6,17 @@ declare(strict_types=1);
 use Magento\Framework\View\Element\Template;
 
 ?>
-<script src="<?= $block->escapeUrl($block->getViewFileUrl('Hyva_Checkout::js/react-checkout.js')) ?>"
-        typeof="text/javascript"
-        async="async"
-        defer="defer">
+<script>
```

Comment:
Curious to know the reason behind of this kind of async loading of script. What is the benefit here?